packages/amplify-category-predictions/src/provider-utils/awscloudformation/triggers/s3/package.json.ejs

```diff
@@ -3,5 +3,8 @@
   "version": "2.0.0",
   "description": "Lambda function generated by Amplify",
   "main": "index.js",
-  "license": "Apache-2.0"
-}
\ No newline at end of file
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@aws-sdk/client-rekognition": "^3.303.0"
+  }
```

Comment:
similarly here, adding SDK to dev dependency is common pattern in lambdas (this package is provided by lambda runtime when deployed).